Dockerfile

```diff
@@ -62,6 +61,17 @@ RUN apk upgrade -U -a && apk add --no-cache \
     unzip \
     yarn
 
+# install s6-overlay, our init system. Workaround for different versions using TARGETPLATFORM
+# platform see https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope
+ARG TARGETPLATFORM
+RUN echo "I am building for $TARGETPLATFORM"
+ARG TARGETPLATFORM
```

Comment:
is it normal to have it a second time here?

Reply:
I don't know actually, but lines 66 and 67 were just to debug it, I suppose they can be removed